app/Config/core.php

```diff
@@ -48,13 +49,13 @@
  *
  * @see ErrorHandler for more information on error handling and configuration.
  */
-	Configure::write('Error', array(
-		'handler' => 'ErrorHandler::handleError',
-		'level' => E_ALL & ~E_DEPRECATED,
-		'trace' => true
-	));
+    Configure::write('Error', array(
+        'handler' => 'ErrorHandler::handleError',
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 31 spaces but found 8